packages/core/src/routes/dashboard.ts

```diff
@@ -1,12 +1,58 @@
+import dayjs, { Dayjs } from 'dayjs';
+
+import { getDailyNewUserCountsByTimeInterval } from '@/queries/log';
 import { countUsers } from '@/queries/user';
 
 import { AuthedRouter } from './types';
 
+const getDateString = (day: Dayjs) => day.format('YYYY-MM-DD');
+
 export default function dashboardRoutes<T extends AuthedRouter>(router: T) {
   router.get('/dashboard/users/total', async (ctx, next) => {
     const { count: totalUserCount } = await countUsers();
     ctx.body = { totalUserCount };
 
     return next();
   });
+
+  router.get('/dashboard/users/new', async (ctx, next) => {
+    const today = dayjs();
+
+    const thirteenDaysAgo = today.subtract(13, 'day');
+    const tomorrow = today.add(1, 'day');
```

Comment:
prefer `today.endOf('day')` and `today.substract(14, 'day').startOf('day')`

Reply:
Refactored. PTAL